modules/core/include/visp3/core/vpMath.h

```diff
@@ -121,6 +122,25 @@ class VISP_EXPORT vpMath
   // combinaison
   static inline long double comb(unsigned int n, unsigned int p);
 
+  /*!
+    Clamp a value to boundaries.
+    \param v : The value to clamp.
+    \param lower, upper : The boundaries to clamp `v` to.
+
+    Throw a vpException if the value of `lower` is greater than `upper`.
+  */
+  template <typename T> static inline T clamp(const T &v, const T &lower, const T &upper)
+  {
+#if (VISP_CXX_STANDARD >= VISP_CXX_STANDARD_17)
+    return std::clamp(v, lower, upper);
```

Comment:
`#include <algorithm>` could be missing for `std::clamp`

Reply:
Already included